### ISSUE TITLE
[FR] In a `gopass create` template, how do I specify that password should not be generated?

### ISSUE THREAD

**Issue by dominikschulz** [gopasspw/gopass#2819]
### Discussed in https://github.com/gopasspw/gopass/discussions/2811

<div type='discussions-op-text'>

<sup>Originally posted by **UtkarshVerma** February 27, 2024</sup>
I am using `gopass` to store my API keys with the following template at `.gopass/create/api-key.yaml`:

```yaml
priority: 1
name: "API Key"
prefix: "api-keys"
name_from:
  - "service"
welcome: "Creating API key entry"
attributes:
  - name: "service"
    type: "string"
    prompt: "Service"
    min: 1
  - name: "password"
    type: "password"
    prompt: "API Key"
```

It works fine, but the problem is that gopass prompts me every time whether a password has to be generated or not. Is there any way I can specify that the password will always be input by the user?

```console
$ gopass create
🌟 Welcome to the secret creation wizard (gopass create)!
🧪 Hint: Use 'gopass edit -c' for more control!
[ 0] Website
[ 1] API Key

Please select the type of secret you would like to create (q to abort) [0]: 1
1
Creating API key entry
  [1] Service                                []: service
  [2] Generate Password?                     [Y/n/q]: n
Enter API Key:
Retype API Key:
❌ Error: the entered password do not match
Enter API Key:
Retype API Key:
✅ Credentials saved to "api-keys/service"
```

In short, I want to skip step [2] and directly ask gopass to let me insert the key.</div>
